src/main/java/org/gelecekbilimde/scienceplatform/post/controller/CategoryController.java

```diff
@@ -32,10 +29,29 @@ SuccessResponse<List<CategoryResponse>> getCategoryList() {
 		return SuccessResponse.success(categoryResponses);
 	}
 
+	@GetMapping("/{id}")
+	SuccessResponse<CategoryResponse> getCategoryById(@PathVariable Long id) {
+		Category category = categoryService.getCategory(id);
+		CategoryResponse categoryResponse = categoryToResponseMapper.map(category);
+		return SuccessResponse.success(categoryResponse);
+	}
+
 	@PostMapping("/create")
```

Comment:
@aleyna-yildizli `POST` HTTP Metodu aslında create ettiğimizi anlatabiliyor bu yüzden davranışı endpointlerde yazmamaya çalışalım. Ek olarak `@RequestMapping`'de tanımlamak yerine aşağıdaki gibi tanımlayabiliriz. Bu sayede listeleme endpotine `/categories` yazabiliriz.
```suggestion
	@PostMapping("/category")
```